lib/static/modules/selectors/tree.js

```diff
@@ -34,7 +34,7 @@ export const getFailedTests = createSelector(
             const browserId = result.parentId;
             const {parentId: testName, name: browserName} = browsers[browserId];
 
-            return {testName, browserName};
+            return {testName, browserName, metaInfo: result.metaInfo};
```

Comment:
In order to not remove `metaInfo` which is not used now inside html-reporter we should write a test.